docs/reqstool/reqstool_config.yml

```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Luftfartsverket/reqstool-client/main/src/reqstool/resources/schemas/v1/reqstool_config.schema.json
+
+type: default
+project_root_dir: "../.."
+locations:
+  annotations: ./dist/annotations.yml
+  test_results_dirs:
```

Comment:
This could possibly change if we are not to put the xml file of the test reports in the dist folder.

Reply:
In [reqstool-python-decorators v0.0.3](https://github.com/Luftfartsverket/reqstool-python-decorators/blob/889632a13600c29a4c180971ac8845359f0a095c/src/reqstool_python_decorators/processors/decorator_processor.py#L187) we changed the path annotations.yml file is saved to.

It should now be build/reqstool/annotations.yml

Reply:
alright! I'll fix